src/index.html

```diff
@@ -1,12 +1,328 @@
+
+          <div class="card__block-stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__block-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__block-price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-total">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__buttom-buy"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 7 Retina 5K (MNED2UA/A)"
+            class="card__image"
+          >
+          <h2 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </h2>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__block-stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__block-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__block-price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-total">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__buttom-buy"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 7 Retina 5K (MNED2UA/A)"
+            class="card__image"
+          >
+          <h2 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </h2>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__block-stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__block-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__block-price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-total">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__buttom-buy"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card">
```

Comment:
Add a blank line between the cards for better readability
```suggestion
        </div>
        
        <div class="card">
```